lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -452,7 +452,7 @@ sub mark_job_linked {
     my $job = $self->find({id => $jobid});
     return undef if !$job || ($referer->path_query =~ /^\/?$/);
     my $comments = $job->comments;
-    return undef if $comments->find({text => {like => 'label:linked%'}});
+    return undef if $comments->search({text => {like => 'label:linked%'}}, {rows => 1})->first;
```

Comment:
When limiting to one row it should be possible to continue using find.

Reply:
`In addition to key, "find" recognizes and applies standard resultset attributes in the same way as "search" does.` [as per docs](https://metacpan.org/pod/DBIx::Class::ResultSet#find)